README.md

```diff
@@ -28,182 +28,4 @@
 
 [![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
-**LATEST RELEASE: You are currently on the main branch which tracks
-under-development progress towards the next release. The current release branch
-is [r23.10](https://github.com/triton-inference-server/vllm_backend/tree/r23.10)
-and which corresponds to the 23.10 container release on
-[NVIDIA GPU Cloud (NGC)](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver).**
-
```

Comment:
We need to minimize our impact on `README.md`

Reply:
Let's have it as is, we may wan't to get rid of those headers **LATEST REASE**

Reply:
Done!

Reply:
That notice is redundant